src/i18n/en.json

```diff
@@ -0,0 +1,20 @@
+{
+	"en": {
+		"Files": "Files",
```

Comment:
I think it would be better to use qualified paths:
```json
{
    "en": {
        "application": {
            "edit": "Edit"
        },
        "editor": {
            "label": "Code Editor"
        }
    }
}
```

And then in the code use qualified paths to reference messages: `Globalize.formatMessage('editor/label')`